homeassistant/components/template/alarm_control_panel.py

```diff
@@ -76,7 +82,38 @@ class TemplateCodeFormat(Enum):
     text = CodeFormat.TEXT
 
 
-ALARM_CONTROL_PANEL_SCHEMA = vol.Schema(
+LEGACY_FIELDS = TEMPLATE_ENTITY_LEGACY_FIELDS | {
+    CONF_VALUE_TEMPLATE: CONF_STATE,
+}
+
+DEFAULT_NAME = "Template Alarm Control Panel"
+
+ALARM_CONTROL_PANEL_SCHEMA = vol.All(
```

Comment:
This is nicely sorted, but please sort the keys in `LEGACY_ALARM_CONTROL_PANEL_SCHEMA` and `ALARM_CONTROL_PANEL_CONFIG_SCHEMA`